src/number_theory/legendre_symbol/mul_character.lean

```diff
@@ -221,6 +221,11 @@ lemma map_zero {R : Type u} [comm_monoid_with_zero R] [nontrivial R] (χ : mul_c
   χ (0 : R) = 0 :=
 by rw [map_nonunit χ not_is_unit_zero]
 
+/-- If the domain is a ring `R`, then `χ (ring_char R) = 0`. -/
+lemma map_ring_char {R : Type u} [comm_ring R] [nontrivial R] (χ : mul_char R R') :
```

Comment:
I think this makes a reasonable simp lemma.

Reply:
The `simp_nf` linter says that `simp` can already prove this via `by simp only [mul_char_class.map_nonunit, ring_char.nat.cast_ring_char, not_is_unit_zero, not_false_iff]` (which is correct).
What should I do? Not declare it `@[simp]` or try to follow the linter's suggestion "try reordering lemmas or adding @[priority]."? But then I've no idea what priority would be reasonable here. Or replace use of the lemma by a suitable `simp only` and remove the lemma?

Reply:
If you want to keep it just don't tag it `simp`.

Reply:
I use it only once, but I think it is a reasonable statement to have. So I'll leave it un-`@[simp]`ed.